### PR TITLE
Ensure that each KCD offers a scholarship

### PR DESCRIPTION
**Pull request by caniszczyk**

### CODE REVIEW

**Comment by jberkus** [comment]
Does the CNCF have advice on how to administer the diversity scholarships for a small event?  Serious question, we're trying to figure it out.

File: .github/ISSUE_TEMPLATE/host.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Please fill in information, organizers, and sponsors as you assemble them, and t
 * [ ] All organizers agree to follow the event guidelines and CNCF Code of Conduct
 * [ ] Minimum of 3 sponsors committed
 * [ ] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
+* [ ] Agree to offer a diversity scholarship for the event
```
Comment (iennae) [inline .github/ISSUE_TEMPLATE/host.md]: For the issue, as a basic requirement this should be 

[ ] Agree to offer 3 complimentary tickets to increase diversity for the event.

Each event could offer additional sponsorship in terms of lodging, airfare, childcare and other expenses, but scholarship isn't very clear and not all events will have the financial support to offer monetary support.

**Comment by iennae** [comment]
> Does the CNCF have advice on how to administer the diversity scholarships for a small event? Serious question, we're trying to figure it out.

@jberkus, that's a great question. We'll work on putting together some recommendations on tickets and sponsorships for increasing diversity at the event.

**Comment by iennae** [comment]
I've added Issue 270 to track efforts for documenting diversity efforts.

**Comment by caniszczyk** [comment]
LGTM

**Pull request merged**